pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -194,16 +236,52 @@ func buildStatefulSet(mdb mdbv1.MongoDB) (appsv1.StatefulSet, error) {
 		},
 	}
 
+	dataVolume, dataVolumeClaim := buildDataVolumeClaim()
+	v := statefulset.CreateVolumeFromConfigMap("automation-config", "example-mongodb-config")
```

Comment:
are we not just using an emptyDir volume for sharing the config file between the side car and the main container?

![](https://media.giphy.com/media/NEJCPYzr6xmAU/giphy.gif)